partners/README.md

```diff
@@ -0,0 +1,108 @@
+# Partner Tekton Task submission process
+
+Partners can submit their Tekton Task in [build-definitions](https://github.com/redhat-appstudio/build-definitions) repository by sending a pull request.
+
+build-definitions repository CI job will validate the PR against a [set of checks](#checks).
+
+build-definitions repository maintainers needs to approve and merge the pull request before the Task can be used.
+
+### How to submit a Task?
+
+Create your Task YAML and put it inside a directory of structure `partners/<task_name>/<task_version>/<task_name>.yaml` and 
+also include a `OWNERS` file inside the Task directory `partners/<task_name>/OWNERS`
+
+For example, if your Task name is `my_task` and the version of the your Task is `0.1`, your Task YAML and OWNERS file location will be `partners/my_task/0.1/my_task.yaml` and `partners/my_task/OWNERS`
+
+Send a pull request to the [build-definitions](https://github.com/redhat-appstudio/build-definitions) repository containing your Task YAML and OWNERS file.
+
+### How to wire the Task with the users pipeline?
```

Comment:
Unfortunatelly this will not work for user. User will do this in user's repository, so it cannot point to  external. Currently we do not provide way how to use partner task.

Reply:
@Michkov Thanks for the input. For now I will remove this section. It would make sense when we have a way to integrate it in future.

Reply:
or we can describe how to push the task into their bundle and how to consume.